src/components/EmbedWidgetModal/components/EmbedWidget/EmbedWidget.tsx

```diff
@@ -71,7 +71,14 @@ const EmbedWidget = () => {
             {t('embed_widget.embed_title')}
           </BodyText>
         </EmbedLabel>
-        <BodyText variant="body2">{t('embed_widget.embed_text')}</BodyText>
+        <BodyText variant="body2">
+          <Trans
+            components={{
+              bold: <strong />,
```

Comment:
styles for `strong` might be overwritten in keen-web, I suggest using `FontBold` from typography instead